### PR TITLE
Extracted base_condition_template from sale_condition_template

### PR DESCRIPTION
**Pull request by guewen**
Extracted from https://github.com/OCA/sale-workflow/pull/50

### CODE REVIEW

**Comment by yvaucher** [comment]
perfect :+1: 

Thanks

File: base_condition_template/condition.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+class BaseConditionTemplate(models.Model):
+    _name = "base.condition.template"
+    _description = "Base condition template text"
+
+    name = fields.Char('Condition summary', required=True)
+    position = fields.Selection([('before_lines', 'Before lines'),
+                                 ('after_lines', 'After lines')],
+                                'Position',
+                                required=True,
+                                help="Position on document")
```
Comment (pedrobaeza) [inline base_condition_template/condition.py:34]: For required fields, you should provide a default value.

Comment (yvaucher) [inline base_condition_template/condition.py:34]: @pedrobaeza What if the field must be defined ?

Takes account_invoice partner_id is not defined by default.

Comment (pedrobaeza) [inline base_condition_template/condition.py:34]: You're right, but in this case, I think we can default to one of them.

Comment (guewen) [inline base_condition_template/condition.py:34]: Changed

**Comment by pedrobaeza** [comment]
Thanks :+1:

**Pull request merged**